src/rpc/server/scala/serv/src/main/scala/com/ctc/omega_edit/grpc/Editors.scala

```diff
@@ -110,7 +110,7 @@ class Editors extends Actor with ActorLogging {
       context.child(id) match {
         case None => sender() ! Err(Status.NOT_FOUND)
         case Some(s) =>
-          s ! PoisonPill
```

Comment:
@scholarsmate Not sure why it worked in CI on Ubuntu but for me on my Mac this wouldn't actually destroy a Akka actor and only doing it this way, `context.system.stop(s)` made it function properly. Thought it would have fixed the issue with Viewport as well but it didn't.